sites/test-site/src/templates/product_listing.jsx

```diff
@@ -24,8 +26,11 @@ import {
 } from '../components/ProductListing';
 import { FilterByGroupSingleSiteWide } from '../components/FilterByGroup';
 
+const SiteInfoHelmet = withMetaSiteInfo(Helmet);
+
 const main = props => (
   <Page {...props}>
+    <SiteInfoHelmet />
```

Comment:
Is this the only page where site info metadata will be available? I thought all pages would have to include this data.

Reply:
moved to default template page according to AC1